ci/requirements-ac-test.txt

```diff
@@ -1,27 +1,27 @@
 atomicwrites==1.3.0
 attrs==19.3.0             # via pytest
 editdistance==0.5.3
-importlib-metadata==1.5.0  # via pluggy, pytest
+importlib-metadata==1.6.0  # via pluggy, pytest
 joblib==0.14.1            # via scikit-learn
 more-itertools==8.2.0     # via pytest
-nibabel==3.0.1
-numpy==1.17.5             # via nibabel, scikit-learn, scipy
-packaging==20.1           # via pytest
+nibabel==3.0.2
+numpy==1.18.2             # via nibabel, scikit-learn, scipy
```

Comment:
looks like I need to update update_dependencies script...

Reply:
What do you mean?

Reply:
this https://github.com/opencv/open_model_zoo/pull/1039/files

Reply:
we still preserve old numpy due to pycocotools win fork... But dependencies were divided to core and extra for support ARM